cli/cmd/resource.go

```diff
@@ -31,17 +29,27 @@ func init() {
 }
 
 func runResource(cmd *cobra.Command, args []string) {
-	URL := viper.GetString("zabbix.url")
+	URL := viper.GetString("zabbix.URL")
```

Comment:
The keys used here can be extracted to constants

Reply:
Ignore this comment about extracted the keys to constants. I thought I removed it.

Reply:
already made them constant in new push. Please review again, and let's see if it is better to just use viper.GetString instead of new function.